### PR TITLE
@kanaabe => Context in tests, partner link

### PR DESCRIPTION
**Pull request by eessex**
- Fixes partner links in artwork captions to not use `/partner/` in url
- Fixes some font errors in caption links
- Adds helper test method `WrapperWithContext` for providing context to components in tests
- Adds helper method `WrapperWithFullscreenContext` for FullscreenViewer context
- Fixes all test that were raising errors over fullscreen issues.

### CODE REVIEW

File: src/Components/Publishing/Fixtures/Helpers.tsx [hunk 1]
```diff
@@ -8,3 +10,40 @@ export const EditableChild = type => {
...
+  viewerIsOpen: PropTypes.bool,
+}
+
+export const WrapperWithFullscreenContext = children => {
+  return WrapperWithContext(
+    FullscreenViewerContext,
+    FullscreenViewerContextTypes,
+    children
+  )
+}
```
Comment (kanaabe) [inline src/Components/Publishing/Fixtures/Helpers.tsx]: 👌

File: src/Components/Publishing/Sections/ArtworkCaption.tsx [hunk 2]
```diff
@@ -152,10 +152,8 @@ export class ArtworkCaption extends React.Component<ArtworkCaptionProps, null> {
       const createTextLink = Boolean(linked && slug)
 
       if (createTextLink) {
-        const href = `/partner/${slug}`
-
         return (
-          <TextLink key={2} href={href} color="#999">
+          <TextLink key={2} href={`/${slug}`} color="#999">
```
Comment (kanaabe) [inline src/Components/Publishing/Sections/ArtworkCaption.tsx]: 🙏

File: src/Components/Publishing/Fixtures/Helpers.tsx [hunk 3]
```diff
@@ -8,3 +10,40 @@ export const EditableChild = type => {
 export const TextFromArticle = article => {
   return compact(map(article.sections, "body")).join("")
 }
+
+export const WrapperWithContext = (context, contextTypes, children) => {
+  const wrapperWithContext = createClass({
```
Comment (damassi) [inline src/Components/Publishing/Fixtures/Helpers.tsx]: Why use `create-react-class`?

Comment (eessex) [reply src/Components/Publishing/Fixtures/Helpers.tsx]: `createClass` was removed from the API in React 16, and it threw a message saying to use this package.  Is there another syntax to achieve this without createClass?

**Pull request merged**